packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -1,71 +1,115 @@
-import React from "react";
-import Downshift from "downshift";
+// @ts-ignore
+import CoreToggle from "@nrk/core-toggle/jsx";
+import React, { useEffect, useState } from "react";
+import { v4 as uuid } from "uuid";
+import { useListNavigation } from "./useListNavigation";
 
 interface Props {
     label: string;
     items: string[];
     className?: string;
-    onChange: (selection: string) => void;
     initialInputValue?: string;
+    onChange?: (value: string) => void;
 }
 
-export const Dropdown = ({ label, items, className = "", initialInputValue, onChange }: Props) => {
-    const onSelectionChange = (selection: string) => onChange(selection);
+interface CoreToggleSelectEvent {
+    detail: { textContent: string };
+    target: { hidden: boolean; button: HTMLButtonElement; value: { textContent: string } };
+}
+
+function lower(value: string | undefined) {
+    return value === undefined
+        ? ""
+        : value
+              .toString()
+              .toLowerCase()
+              .replace(/\s/g, "");
+}
+
+function focusSelected(listEl: HTMLElement, listId: string, selected: string | undefined) {
+    let focusedItem: HTMLElement | null;
+    if (selected !== undefined) {
+        focusedItem = listEl.querySelector(`#${listId}__${lower(selected)}`);
+    } else {
+        focusedItem = listEl.querySelector('[role="option"]');
+    }
+    focusedItem && focusedItem.focus();
+}
+
+export const Dropdown = ({ items, initialInputValue, label, onChange }: Props) => {
+    const [selectedValue, setSelectedValue] = useState(initialInputValue);
+
+    useEffect(() => {
```

Comment:
Does this actually do something? SelectedValue is set to initialInput in contruction. 

So if this actually should do something, initialInputValue is probably not the correct name

Reply:
This is Simen's work, but I think this is because of the web component loading on first render, and changing `selectedValue` back to empty/`undefined`. Therefore, the `selectedValue` is updated again after rendering the first time, to ensure the correct value is displayed to the user.